drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -285,7 +285,7 @@ int bnxt_hwrm_set_filter(struct bnxt *bp,
 
 	enables = filter->enables |
 	      HWRM_CFA_L2_FILTER_ALLOC_INPUT_ENABLES_DST_ID;
-	req.dst_id = rte_cpu_to_le_16(vnic->fw_vnic_id);
+	req.dst_id = rte_cpu_to_le_16(dst_id);
```

Comment:
This is essentially the same. Right?
Previously we were passing vnic, but just dereferencing fw_vnic_id from that.
Now we pass just the vnic->fw_vnic_id.